src/cpp/cpp_typecheck_compound_type.cpp

```diff
@@ -11,6 +11,12 @@ Author: Daniel Kroening, kroening@cs.cmu.edu
 
 #include "cpp_typecheck.h"
 
+//#define DEBUG
```

Comment:
Nit: don't check this line in ( @peterschrammel )

Reply:
Agreed (also with the other files where this occurs).

Reply:
These are already fixed in the latest commit